core/templates/pages/contributor-dashboard-page/services/translate-text.service.ts

```diff
@@ -85,6 +123,9 @@ angular.module('oppia').factory('TranslateTextService', [
               stateWiseContents[stateName])) {
               if (text !== '') {
                 contentIds.push(contentId);
+                stateAndContent.push(
+                  new StateAndContent(
+                    stateName, contentId, text as string));
```

Comment:
Why is it necessary to cast `text as string`?

Reply:
When text is extracted from `Object.entries(stateWiseContents[stateName])` it is recognized as unknown instead of string so I needed to cast it here.

Reply:
Ah, gotcha. Can you add a comment explaining this?